core/geometry/src/test/SimpleFactory.ts

```diff
@@ -213,7 +213,7 @@ export class SimpleFactory {
         return g.Matrix4d.createIdentity();
       case 2:
         return g.Matrix4d.createTranslationAndScaleXYZ(1, 3, 2, 3, 2, 4);
-      case 2:
+      case 3:
```

Comment:
@saeeedtorabi @dassaf4 heads up, vitest emitted a warning that this case was previously unreachable, so I updated it. No existing test failed

Reply:
Nice, wtg vitest!